contracts/Comptroller.sol

```diff
@@ -112,9 +112,14 @@ contract Comptroller is ComptrollerV7Storage, ComptrollerInterfaceG2, Comptrolle
         admin = msg.sender;
     }
 
-    modifier onlyProtocolAllowed {
```

Comment:
what is the reasoning behind changing this modifier into a function ?

Reply:
Comptroller size. The current comptroller in the core pool is too close to the size limit, and modifiers occupy lots of space because they are copied verbatim into every function (instead of a bare JUMP for private method calls)

Reply:
Thanks